### PR TITLE
[OR-856] add @tokamak-network/titan-contracts

### PR DESCRIPTION
**Pull request by boohyung**
added @tokamak-network/titan-contracts in packages/tokamak
* change the path
* check the dependencies
* change the target for CI/CD and docker
* add @tokamak-network/titan-contracts in github actions workflow
* remove @eth-optimism/contracts

I have finished some testing for this work below.
* build the docker image
* run the docker image
* coverage test (unit-test)
* integration test

### CODE REVIEW

File: packages/contracts/contracts/L2/predeploys/WETH9.sol [hunk 1]
```diff
@@ -16,53 +16,56 @@
 pragma solidity >=0.4.22 <0.6;
 
 contract WETH9 {
-    string public name     = "Wrapped Ether";
-    string public symbol   = "WETH";
-    uint8  public decimals = 18;
+    string public name = "Wrapped Ether";
```
Comment (rlgns98kr) [inline packages/contracts/contracts/L2/predeploys/WETH9.sol:19]: linter 가 자동으로 바꾼 부분인 것 같은데 추후, optimism update 시 충돌가능성이 있어보입니다.
원래대로 돌려놓는 것이 좋아보입니다!

**Comment by boohyung** [comment]
[84cad69](https://github.com/tokamak-network/tokamak-titan/pull/127/commits/84cad69a738190f7f0892a7704736f51445f30cb)
github-hosted virtual machine have low spec to test coverage (ubuntu-latest: 2 Core, 8GB RAM). so I excluded @eth-optimism/contracts. 
* I have checked stucking test coverage in EC2 (spec: t2.large)

File: packages/contracts/contracts/L2/predeploys/WETH9.sol (hunk 1 above)
Comment (boohyung) [reply packages/contracts/contracts/L2/predeploys/WETH9.sol]: Done. [5ac1166](https://github.com/tokamak-network/tokamak-titan/pull/127/commits/5ac1166ea945c96433b1faf335b2faeebd7f7cf5)

File: packages/contracts/contracts/L2/predeploys/WETH9.sol
Comment (rlgns98kr) [inline packages/contracts/contracts/L2/predeploys/WETH9.sol:1]: 줄하나 추가 부탁드리겠습니다! ㅎㅎ

File: .github/workflows/node-packages.yml [hunk 2]
```diff
@@ -98,7 +98,7 @@ jobs:
         run: yarn build
 
       - name: Test Coverage
-        run: yarn test:coverage
```
Comment (rlgns98kr) [inline .github/workflows/node-packages.yml:101]: 필요없는 코드 같습니다.

**Review by rlgns98kr — APPROVED** [review]
build 및 docker-build 모두 확인하였습니다.

감사드립니다!

**Pull request merged**